BUILD.bazel

```diff
@@ -1 +1,15 @@
 # TODO(#1532): Rename file to 'BUILD' post-Gradle.
+'''
+Oppia Android binary
+'''
+android_binary(
+    name = "oppia",
+    custom_package = "org.oppia.app",
+    enable_data_binding = True,
+    manifest = "//app:src/main/AndroidManifest.xml",
+    multidex = "native",
```

Comment:
I know this was changed when debugging, but prefer to use ``legacy`` since we want to keep multidex support on pre-L devices.

Reply:
Done